codechecker_common/requirements_py/dev/requirements.txt

```diff
@@ -1,4 +1,3 @@
-nose==1.3.7
```

Comment:
I believe, this works because pytest is installed via the `venv_dev` of analysis and web modules. But shouldn't we add pytest in this requirements.txt explicitly?

Reply:
`codechecker_common` didn't use `nosetests` at all, and doesn't use `pytest` either.